src/test/java/core/basesyntax/service/impl/ReaderServiceImplTest.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.ReaderService;
+import java.io.File;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class ReaderServiceImplTest {
+    private static final String VALID_FILE = "src/test/resources/test.csv";
+    private static final String NON_EXISTENT_PATH = "src/test/resources/test123123.csv";
+    private static final String HEADER = "type,fruit,quantity";
+    private static final String VALID_LINE = "s,apple,55";
+    private static ReaderService reader = new ReaderServiceImpl();
```

Comment:
you create it in `beforeClass()` method
```suggestion
    private static ReaderService reader;
```